bsp/Inc/I2C.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2018-2023 UT Longhorn Racing Solar */
+// BSP I2C header file
+

```

Comment:
forgot to ask this but has this been tested on the l4 and f4?

Reply:
Has been tested on F4